consensus/ethash/ethash.go

```diff
@@ -792,3 +792,13 @@ func (ethash *Ethash) APIs(chain consensus.ChainHeaderReader) []rpc.API {
 func SeedHash(epoch uint64, epochLength uint64) []byte {
 	return seedHash(epoch, epochLength)
 }
+
+// CalcEpochLength returns the epoch length for a given block number (ECIP-1099)
+func CalcEpochLength(block uint64, ecip1099FBlock *uint64) uint64 {
```

Comment:
Do you think `GetEpochLength` fits better in this context?

Reply:
IMO, it probably does... but `Calc` echos the pre-existing function `calcEpochLength`. Renaming the functions would add diffs for trivial benefits; I'd suggest postponing nice-to-haves like this till later.